shared/protos/Room.proto

```diff
@@ -107,6 +107,17 @@ message Room {
 
   repeated Tile tiles = 18 [(gmx) = "tiles/tile"];
 
+  message Layer {
+    optional int32 depth = 1;
+    optional string name = 2;
+    optional bool visible = 3;
+
+    repeated Instance instances = 17;
```

Comment:
Why 17?

Reply:
I wasn't sure what fields we were jamming in here yet. I've defragged it for now but before alpha or whatever we may change it.